src/package.xml

```diff
@@ -794,6 +809,7 @@
         <members>Account-HEDA Organization Layout</members>
         <members>Address__c-HEDA Address Layout</members>
         <members>Affiliation__c-HEDA Affiliation Layout</members>
+        <members>Attendance_Event__c-HEDA Attendance Event Layout</members>
```

Comment:
I know this has been in progress since pre-EDA, but similar to the other PR this should be EEEEEEDA :)